sonar-plugin-api/src/main/java/org/sonar/api/profiles/ProfileDefinition.java

```diff
@@ -30,6 +31,7 @@
  * @since 2.3
  */
 @ServerSide
+@ComputeEngineSide
 @ExtensionPoint
 public abstract class ProfileDefinition {
```

Comment:
Initially added it because it's in the API.

I don't know that part, so I'll blindly trust you on this and will remove it